File: time of day.py
# EG 7th time of day

time = int(input("what time is it"))
if time >=17 and time <= 24:
    print("good evening")
elif time <= 11 and time > 0:
    print("good morning")
elif time >= 12 and time <=16:
    print('good afternoon')
else:
    print('please enter a valid time')

File: time-of-day.c

// eg 7th time of day

#include <stdio.h>

int main() {
    int hour;

    // Ask user to enter the hour in military time
    printf("Enter the current hour (0–23): ");
    scanf("%d", &hour);

    // Greet based on input hour
    if (hour >= 18)
        printf("Good evening!\n");
    else if (hour >= 12)
        printf("Good afternoon!\n");
    else
        printf("Good morning!\n");

    return 0;
}